src/dodal/devices/webcam.py

```diff
@@ -3,18 +3,17 @@
 import aiofiles
 from aiohttp import ClientSession
 from bluesky.protocols import Triggerable
-from ophyd_async.core import AsyncStatus, StandardReadable
+from ophyd_async.core import AsyncStatus, StandardReadable, soft_signal_rw
 
-from dodal.devices.ophyd_async_utils import create_soft_signal_rw
 from dodal.log import LOGGER
 
 
 class Webcam(StandardReadable, Triggerable):
     def __init__(self, name, prefix, url):
         self.url = url
-        self.filename = create_soft_signal_rw(str, "filename", "webcam")
-        self.directory = create_soft_signal_rw(str, "directory", "webcam")
-        self.last_saved_path = create_soft_signal_rw(str, "last_saved_path", "webcam")
+        self.filename = soft_signal_rw(str, "filename", "webcam")
+        self.directory = soft_signal_rw(str, "directory", "webcam")
+        self.last_saved_path = soft_signal_rw(str, "last_saved_path", "webcam")
```

Comment:
```suggestion
        self.filename = soft_signal_rw(str, name="filename")
        self.directory = soft_signal_rw(str, name="directory")
        self.last_saved_path = soft_signal_rw(str, name="last_saved_path")
```

`soft_signal_rw` has different arguments to `create_soft_signal_rw`.

We don't need to provide the source prefix to `soft_signal_rw`

Reply:
Concerning that that didn't make any tests fail...

Reply:
Slightly, I'm not overly surprised that the initial_value isn't covered in tests. I don't think that matters for quite a few signals. We should probably cover the names better though.

It's also concerning that the type checker doesn't catch `soft_signal_rw(int, initial_value="string")` as a problem.

Reply:
Hmm, I did this in part to use #497 because the initial values weren't working properly in my tests for the `Undulator`, and I decided to upgrade before tackling the problem. Is it possible that `initial_value` never worked?